src/main/deploy/shooterData.csv

```diff
@@ -1,10 +1,13 @@
 Distance (inches),Angle (degrees),Velocity (RPM)
 Actual point(s)
-60,25,2000
-81,32,2000
-110,38,2100
-142,42,2200
-183,50,2600
-220,53,3000
-300,58,3700
-332,60,4200
\ No newline at end of file
+80,15.77,2375 //change
```

Comment:
Will this comment at the end cause a problem with the csv loader?

Reply:
For DCMP we could consider a confidence level in this chart that can be used to show a green or red (and possibly yellow) box in shuffleboard to signal the drive team that it should get a bit closer before shooting.

Reply:
Currently the CSV parser only handles lines starting with `'#'`, I added support for `//` and comments at the ends of lines in #136, though.